spec/lib/smb2/packet/create_request_spec.rb

```diff
@@ -17,15 +17,19 @@
       ].pack('H*')
     end
 
+    it_behaves_like "packet"
     it_behaves_like "request", Smb2::COMMANDS[:CREATE]
 
     specify 'body' do
       expect(packet.struct_size).to eq(57)
       expect(packet.oplock).to eq(0)
+      expect(packet.security_flags).to eq(0)
       expect(packet.impersonation).to eq(2)
-      expect(packet.access_mask).to eq(0)
       expect(packet.create_flags).to eq(0)
 
+      expect(packet.reserved).to eq(0)
+
+      expect(packet.desired_access).to eq(0x0012_019f)
       expect(packet.file_attributes).to eq(0)
       # SHARE_DELETE | SHARE_WRITE | SHARE_READ
```

Comment:
Comments like this are also helpful to fix [this](https://github.com/rapid7/smb2/pull/28/files#r28821889)